locales-ar.xml

```diff
@@ -204,70 +204,70 @@
-      <single>illustrator</single>
-      <multiple>illustrators</multiple>
+      <single>مرسم</single>
+      <multiple>مرسمون</multiple>
     </term>
     <term name="translator">
       <single>مترجم</single>
-      <multiple>مترجمين</multiple>
+      <multiple>مترجمون</multiple>
     </term>
     <term name="editortranslator">
       <single>مترجم ومحرر</single>
-      <multiple>مترجمين ومحررين</multiple>
+      <multiple>مترجمون ومحررون</multiple>
     </term>
 
     <!-- SHORT ROLE FORMS -->
     <term name="director" form="short">
-      <single>dir.</single>
-      <multiple>dirs.</multiple>
+      <single>مدور</single>
+      <multiple>مدورون</multiple>
     </term>
     <term name="editor" form="short">
       <single>محرر</single>
-      <multiple>محررين</multiple>
+      <multiple>محررون</multiple>
     </term>
     <term name="editorial-director" form="short">
       <single>مشرف على الطبعة</single>
-      <multiple>مشرفين على الطبعة</multiple>
+      <multiple>مشرفون على الطبعة</multiple>
     </term>
     <term name="illustrator" form="short">
-      <single>ill.</single>
-      <multiple>ills.</multiple>
+      <single>مرسم</single>
+      <multiple>مرسمون</multiple>
     </term>
     <term name="translator" form="short">
       <single>مترجم</single>
-      <multiple>مترجمين</multiple>
+      <multiple>مترجمون</multiple>
     </term>
     <term name="editortranslator" form="short">
       <single>مترجم ومشرف على الطباعه</single>
-      <multiple>مترجمين ومشرفين على الطباعه</multiple>
+      <multiple>مترجمون ومشرفون على الطباعه</multiple>
     </term>
 
     <!-- VERB ROLE FORMS -->
-    <term name="container-author" form="verb"></term>
-    <term name="director" form="verb">directed by</term>
+    <term name="container-author" form="verb">انشاء</term>
+    <term name="director" form="verb">اشراف</term>
     <term name="editor" form="verb">تحرير</term>
     <term name="editorial-director" form="verb">اعداد</term>
-    <term name="illustrator" form="verb">illustrated by</term>
+    <term name="illustrator" form="verb">ترسيم</term>
     <term name="interviewer" form="verb">مقابلة بواسطة</term>
     <term name="recipient" form="verb">مرسل الى</term>
-    <term name="reviewed-author" form="verb">بـ</term>
+    <term name="reviewed-author" form="verb">مراجعة</term>
     <term name="translator" form="verb">ترجمة</term>
     <term name="editortranslator" form="verb">اعداد وترجمة</term>
 
     <!-- SHORT VERB ROLE FORMS -->
-    <term name="director" form="verb-short">dir.</term>
+    <term name="director" form="verb-short">اشراف</term>
     <term name="editor" form="verb-short">تحرير</term>
     <term name="editorial-director" form="verb-short">اشرف على الطبعة</term>
-    <term name="illustrator" form="verb-short">illus.</term>
+    <term name="illustrator" form="verb-short">ترسيم.</term>
     <term name="translator" form="verb-short">ترجمة</term>
-    <term name="editortranslator" form="verb-short">ترجمه واشرف على الطباعه</term>
```

Comment:
I don't agree. the original translation is better than the suggested one.

Reply:
I cant see what you prefer.. could you please show it here?

Reply:
This might be a mistake by @gotoayman, since he agreed with the proposed change above at https://github.com/citation-style-language/locales/pull/168#discussion_r158527712. (@gotoayman, it's clear to you that the lines in red will be replaced by the lines in green?)